tests/test_dle.py

```diff
@@ -323,7 +323,7 @@ def test_cache_blocking_imperfect_nest_v2(blockinner):
     shape = (16, 16, 16)
     grid = Grid(shape=shape, dtype=np.float64)
 
-    u = TimeFunction(name='u', grid=grid, space_order=2)
+    u = TimeFunction(name='u', grid=grid, space_order=4)
```

Comment:
why the order increase?

Reply:
due to the new cost model, which is now not allocating temporaries if the reduction in operation count isn't sufficient according to the cost model heuristics. With so=4 we have larger expressions, so CIRE does its job